backend/main.go

```diff
@@ -232,6 +232,8 @@ func initialize(servicePort, databaseURL, redisURL, loggingURL, statsdURL string
 	r.Mount("/affiliate", r.Group(func(r chi.Router) {
```

Comment:
I read in your [discord message](https://discord.com/channels/880194069702262855/885275622287478815/1098740091838550076) to @ejrcarr that r.Group doesn't work? What I meant is that we should just _remove_ `r.Mount("/affiliate")` and keep the `r.Group`. With that, everything within the group will be using the `CheckIdentity` middleware without being at the "/affiliate" level.

```suggestion
	r.Group(func(r chi.Router) {
        .
        .
        .
    })
```

Reply:
This has been fixed.